### ISSUE TITLE
Determine Conventions

### ISSUE THREAD

**Issue by oliver-pham** [tuanthanh2067/seeksi#9]
There are conventions to enhance communication, collaboration, and productivity. Before we start developing our project, I think we should decide on a few conventions below:

- [ ] Commit message
- [ ] Branch name
- [x] #12 
- [x] #8 

I think [these naming conventions](https://namingconvention.org/git/) are a great start. Maybe we can use them.

Feel free to create a new issue and suggest a convention for each item. This issue is used for tracking all the conventions to be implemented.

## Resources:
- https://namingconvention.org/
